File: compiler/GENERATOR/JAVA/TEST/innerclasses/helloworld.java
import java.lang.*;

class HelloWorld {

    public String print(){
	return "Hello " + new World().print();
    }

    public class World {
	public String print(){
	    return "World "; 
	}
    }

    public static void main(String[] args) {
	System.out.println(new HelloWorld().print());
    }
}

File: dotnetlib/dotnet2beta/test/pinvoke/helloworld.cs
/* 
   cl -Fehellolib.dll -Zi -LD hellolib.c -link -DEF:hellolib.def
   csc -t:library hellowrapper.cs
   csc -t:exe -r:hellowrapper.dll helloworld.cs
   helloworld.exe
 */

using System;

public class helloworld
  {
    public static void Main(String[] args)
      {  
	hellolib.hello("C#");
      }
  }
